plugins/secretstores/jwt/sample.conf

```diff
@@ -0,0 +1,7 @@
+# JWT Secret Store
+[[secretstores.jwt]]
+  id = "my_jwt_generator"
+  dynamic = false
+  users = ["root", "root"]
+  urls = ["http://127.0.0.1:8007/data_collection/api/v1/token/", "http://127.0.0.1:8007/data_collection/api/v1/token/"]
+  passwords = ["root", "root"]
```

Comment:
Please comment the options here and be more verbose. Furthermore, sensible default values should be commented. Additionally I think you should make each `url` an own section... Here my suggestion

```suggestion
# JSON Web Tokens secret-store
[[secretstores.jwt]]
  ## Unique identifier for the secret-store.
  ## This id can later be used in plugins to reference the secrets
  ## in this secret-store via @{<id>:<secret_key>} (mandatory)
  id = "secretstore"

  ## URL to query for tokens
  ## This URL will be used for all tokens defined below with the corresponding user credentials.
  url = "http://127.0.0.1:8007/data_collection/api/v1/token/"

  ## Definition of a token generated by the endpoint
  ## Multiple token sections can be used to retrieve different tokens from the endpoint.
  [[secretstores.jwt.token]]
    ## Key for the later referencing the secret via @{<id>:<secret_key>} (mandatory)
    key = "mysecret"
    
    ## Username and password for retrieving the token
    user = "root"
    password = "s3cr3T"
```